OSDOCS-16939: DITA split-modules — olm packaging format (#115280)
openshift/openshift-docs: modules/olm-bundle-format.adoc

//
// * operators/understanding/olm-packaging-format.adoc

:_mod-docs-content-type: CONCEPT
[id="olm-bundle-format_{context}"]
= Bundle format

[role="_abstract"]

> The bundle format is an Operator Framework packaging format that simplifies distributing Operator metadata for catalogs in {product-title}. An Operator bundle is a single Operator version shipped as a non-runnable container image that stores Kubernetes manifests and metadata.

Reviewer: ```suggestion
The bundle format is an Operator Framework packaging format that simplifies distributing Operator metadata to catalogs. An Operator bundle is a single Operator version shipped as a non-runnable container image that stores Kubernetes manifests and metadata.
```
